src/Arcus.Messaging.AzureFunctions.ServiceBus/Arcus.Messaging.AzureFunctions.ServiceBus.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Should this also be targetting .NET 6 ?

Reply:
For another PR, let's focus on this major feature first. We can then move all projects in one go.

Reply:
We otherwise have to change all the other projects, the CI build...